src/explore-education-statistics-common/src/components/AccordionSection.tsx

```diff
@@ -81,7 +81,17 @@ const AccordionSection = ({
             headingTag,
             {
               className: classes.sectionHeading,
-              id: heading.toLowerCase().split(' ').join('-'),
+              id: heading
+                .toLowerCase()
+                .split(' ')
+                .join('-')
+                /* 
+                workaround for bad IDs that start with a number from 
+                being generated which throws the following error 
+                DOMException: Failed to execute 'querySelector' on 'Element': '#6.test-title' is not a valid selector.
+              */
+                .replace(/[0-9]/, 'section-')
+                .replace(/\./g, '-'),
```

Comment:
Not a big fan of this. The above will give weird results if the heading contains numbers in non-starting positions e.g. 'Some title with 5 words' would give us `some-title-with-section-words`.

As mentioned in standup, we could just use Lodash's `kebabCase` to do this in a nice clean way:

```ts
id: `section-${kebabCase(heading)}`
```

Note that to avoid the id starting with a number, we just need to prefix the id with something generic like `section-`

Reply:
thanks, was quite bit distracted yesterday so forgot about the lodash solution. Looks like this is a cleaner fix & works much better than trying to write the perfect regex for this